cpp-client/deephaven/client/include/private/deephaven/client/server/server.h

```diff
@@ -67,6 +67,63 @@ struct ServerResponseHolder final : public CompletionQueueCallback {
   Response response_;
 };
 
+// Options for the creation of the underlying grpc channel.
+class ClientOptions {
+public:
+  typedef std::vector<std::pair<std::string, int>> int_options_t;
+  typedef std::vector<std::pair<std::string, std::string>> string_options_t;
+public:
+  static ClientOptions defaults() {
+    return ClientOptions();
+  }
+
+  ClientOptions &setUseTls(const bool useTls) {
+    useTls_ = useTls;
+    return *this;
+  }
+
+  ClientOptions &setPem(const std::string &pem) {  // root certificate to use
+    pem_ = pem;
+    return *this;
+  }
```

Comment:
Prefer having doxygen comments and using `std::move` as in

~~~
  /**
   * Sets the root certificate.
   * @param pem root certificate to use
   * @return *this, to be used for chaining.
   */
  ClientOptions &setPem(std::string pem) {
    pem_ = std::move(pem);
    return *this;
  }
~~~

Reply:
This is a private header file.  There are no doxygen comments anywhere in this file.

Reply:
That is a good point. However (and this is just a proposal) I think I would like to migrate to doxygen style everywhere, even for private header files. This is because my IDE renders them (when you hit control-Q on a symbol) which I think is great and looks nice.

So I think I might like to propose that, even for private header files, if there's going to be a comment provided, maybe it should be a doxygen-style comment.